content/doc/developer/development-environment/index.adoc

```diff
@@ -0,0 +1,24 @@
+---
+layout: developerchapter
+wip: true
+summary: Preparing your development environment for Jenkins core and plugin development
+references:
+- url: https://wiki.jenkins-ci.org/display/JENKINS/IntelliJ+IDEA+plugin+for+Stapler
+  title: IntelliJ IDEA plugin for Stapler
+- url: https://wiki.jenkins-ci.org/display/JENKINS/NetBeans+plugin+for+Stapler
+  title: NetBeans plugin for Stapler
+- url: https://wiki.jenkins-ci.org/display/JENKINS/Setting+up+Eclipse+to+build+Jenkins
+  title: Setting up Eclipse to build Jenkins
+- url: https://wiki.jenkins-ci.org/display/JENKINS/Jenkins+plugin+development+with+Eclipse
+  title: Jenkins plugin development with Eclipse
+- url: https://wiki.jenkins-ci.org/display/JENKINS/Eclipse+alternative+build+setup
+  title: Eclipse alternative build setup
+- url: https://wiki.jenkins-ci.org/display/JENKINS/Developing+with+JRebel
+  title: Developing with JRebel # TODO find someone who can write actual documentation for this
+- url: https://wiki.jenkins-ci.org/display/JENKINS/Using+Artifactory+for+Configuring+a+Jenkins+Build+Environment
+  title: Using Artifactory for Configuring a Jenkins Build Environment # TODO really?
+- url: https://wiki.jenkins-ci.org/display/JENKINS/Jenkins+Development+Environment+with+Nexus
+  title: Jenkins Development Environment with Nexus # TODO really?
```

Comment:
Killing for me in this context means to also remove the wiki page, and that was out of scope for now. Or are you saying a standalone wiki page nobody cares about should be left around?